packages/manager/src/features/CloudPulse/Alerts/Utils/AlertsActionMenu.ts

```diff
@@ -33,12 +33,17 @@ export const getAlertTypeToActionsList = (
       title: 'Edit',
     },
     {
+      disabled: isStatusActionDisabled(alertStatus),
```

Comment:
```suggestion
      disabled: (alertStatus === 'in progress' || alertStatus === 'failed'),
```

no need to have a function for this